client/src/components/StartAnimation.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import leo1 from '../assets/leo1.svg';
+import leo2 from '../assets/leo2.svg';
+import { Foos, Buddy } from '../components/Title';
+
+export const AnimationContainer = styled.div`
+  width: 100%;
```

Comment:
Maybe change these to `100vw` and `100vh` respectively.

Reply:
I tried it before with vh, but it did not take up the whole screen.